src/backend/web/routes/feeds.js

```diff
@@ -51,4 +52,19 @@ feeds.get('/:id', async (req, res) => {
   }
 });
 
+feeds.post('/', protect, async (req, res) => {
+  const { newFeed, email } = req.body;
+  try {
+    /* check if email has a key in redis
+  if not create key
+  if user has a key in redis, check if feed already exists, (not sure how fast this is? or if its necessary), if not store feed in feeds.
+  */
+  } catch (err) {
+    logger.error({ err }, 'Unable to add feed to redis');
```

Comment:
We're using `(error)` instead of  `(err)`

Reply:
okay cool, ill fix accordingly.